pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -105,10 +92,17 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 	if err != nil {
```

Comment:
Good catch!  We did that elsewhere, but not here.

@madhusudancs Please submit a follow-up PR to do that.